package.json

```diff
@@ -8,24 +8,22 @@
     "url": "https://github.com/latticexyz/mud.git"
   },
   "scripts": {
+    "// TODO make this use turob instead?  an we get rid of lerna?  link:packages": "lerna exec 'pnpm link --global'",
```

Comment:
no need for this anymore since we have `mud set-version --link path/to/mud` to link to a local version of MUD now